### ISSUE TITLE
Plans on supporting STJ (instead of Newtonsoft.JSON)?

### ISSUE THREAD

**Issue by angelaki** [apimatic/core-lib-csharp#82]
I just found this project since the PayPal API obviously uses it (https://github.com/paypal/PayPal-Dotnet-Server-SDK). Since I cannot use Newtonsoft because of version mismatches with other dependencies I wanted to ask if you plan to support STJ (anytime soon).

I already asked in their repo what is obviously wrong (https://github.com/paypal/PayPal-Dotnet-Server-SDK/issues/13).

**Comment by hamzamahmood** [comment]
Thank you for reaching out. We understand the challenges version mismatches can bring. While migrating to `System.Text.Json` is on our roadmap, it will take some time to implement. In the meantime, could you please share the version details of the `Newtonsoft.Json` packages currently in your project? This will help us diagnose and potentially address the version conflict you're encountering.

Looking forward to your response!

**Comment by angelaki** [comment]
In this case it's actually just a mismatch between https://github.com/paypal/PayPal-Dotnet-Server-SDK and https://github.com/twilio/twilio-csharp. After referencing Newtonsoft in my executable directly it works but ... you know, anyway I'd like to reduce the number of dependencies. And after migration my whole project to STJ I noticed how many libs still depend on Newtonsoft.

**Comment by hamzamahmood** [comment]
@angelaki we've released a patch version of the `APIMatic.Core` package that now uses `Newtonsoft.Json` `>= 13.0.1` instead of `>=13.0.2`, allowing for compatibility with a wider range of libraries. To resolve the version conflict, you will need to wait for the next version bump of `PayPalServerSDK` which will include the updated version of `Apimatic.Core`.

Updating our `Apimatic.Core` package to use `System.Text.Json` is on our roadmap, but we've made this release to help you in the immediate term.
